broccoli-template-linter.js

```diff
@@ -101,15 +111,21 @@ TemplateLinter.prototype.processString = function(contents, relativePath) {
   }, this)
         .join('\n');
 
-
-  let output = this._generateTestFile(
-    'TemplateLint - ' + relativePath,
-    [{
-      name: 'should pass TemplateLint',
-      passed: passed,
-      errorMessage: jsStringEscape(relativePath + ' should pass TemplateLint.\n' + errorDisplay)
-    }]
-  );
+  let output = '';
+  if (this._testGenerator) {
```

Comment:
> If you have a project today without ember-cli-qunit or ember-cli-mocha (prior to this PR) what happens?

Ember CLI will display a warning that you should install a test framework (for every generated test 😞) and it will generate an empty file

> What happens after this change?

`ember-cli-template-lint` will display a warning that it couldn't detect a test framework (once) and it will generate an empty file